src/MEDS_DEV/datasets/template/README.md

```
@@ -0,0 +1,24 @@
+- **Required training**: Specify any training or certification required to access the dataset.
+
+## Supported Tasks
+Describe the existing tasks already present in MEDS-DEV that are covered. If there are new tasks that can be added, describe them here. Also note the `predicates.yaml` file that specifies the dataset's predicates.
+
+## MEDS-transformation
+Shortly specify the process of transforming this dataset to the MEDS format. If the dataset is already in the MEDS format when downloaded, specify that here.
+
+## Sources
+
+Summarize the sources of the dataset. If the dataset is a combination of multiple sources, list them here.
+1. https://link-to-dataset.org
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)